test/compilable/testCtfeWriteln.d

```diff
@@ -0,0 +1,30 @@
+/*
+TEST OUTPUT:
+0^^2 == 1
+1^^2 == 1
+2^^2 == 4
+3^^2 == 9
+4^^2 == 16
+5^^2 == 25
+6^^2 == 36
+result == 91
+*/
+int sum_of_sq(int x) pure nothrow @safe
+{
+    int result = 0;
+    foreach (i; 0 .. x)
+    {
+        import std.conv;
```

Comment:
I believe this restriction may have been lifted (or wasn't properly enforced), other test modules have imports to phobos as well.